src/Microsoft.Data.Analysis/DataFrame.BinaryOperators.cs

```diff
@@ -32,16 +22,6 @@ public partial class DataFrame
             return df.ReverseAdd(value);
         }
 
-        public static DataFrame operator +(DataFrame df, char value)
```

Comment:
For conversational sake - could this be useful for appending a character to string columns? Or would they just use a full string in that case?

Reply:
2 things:
1. This operator is on the DataFrame itself. So it'd be useful only if we had a DataFrame composed of only char columns. Even then, the current behavior is not great: 'a' + 'a' results in whatever character `194` is. 
2. Assuming you mean the same operator but defined on the `DataFrameColumn` type, then no, it wouldn't be useful. We'd just use the full `StringDataFrameColumn` in that case. We don't have the ability to go from a `PrimitiveDataFrameColumn<char>` -> `StringDataFrameColumn` yet.